tests/filecheck/dialects/irdl/testd.irdl.mlir

```diff
@@ -144,5 +144,39 @@ builtin.module {
       irdl.operands(%0, single %0, optional %0, variadic %0)
       irdl.results(%0, single %0, optional %0, variadic %0)
     }
+
+    // CHECK:      irdl.operation @op_with_regions {
+    // CHECK-NEXT:   %{{.*}} = irdl.region
+    // CHECK-NEXT:   %{{.*}} = irdl.region()
+    // CHECK-NEXT:   %{{.*}} = irdl.is i32
+    // CHECK-NEXT:   %{{.*}} = irdl.is i64
+    // CHECK-NEXT:   %{{.*}} = irdl.region(%{{.*}}, %{{.*}})
+    // CHECK-NEXT:   %{{.*}} = irdl.region with size 3
+    // CHECK-NEXT:   irdl.regions(%{{.*}}, %{{.*}}, %{{.*}}, %{{.*}})
```

Comment:
why anonymize?

Reply:
thought it was the done thing

Reply:
Yes and no, we don't want spurious changes in the filecheck tests if we add something in the middle, but if you print the IR you'll see that the names of the values are preserved, so it will be stable, and they're actually really useful in this case to track how the constraints are propagated, I'd recommend deanonymizing. I added an option to `filecheckize` to do this: `--xdsl-anonymize`, this will only anonymize `%[\d]+`, not ssa values with name hints.

Reply:
`cat my_ir.mlir | filecheckize`

Reply:
and where would one get filecheckize from?

Reply:
`pip`, as usual :)

Reply:
Ah I was trying to find it as a subproject of xdsl